docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -489,6 +477,19 @@ Reference](../../../../org.tizen.native.mobile.apireference/index.html).
 | `http://tizen.org/feature/watch_app`     | Specify this key, if the application requires the watch application feature. | 4.0   |
 | `http://tizen.org/feature/multimedia.media_codec`     | Specify this key, if the application requires the media codec feature. | 4.0   |
 
+**Table: Available IoT-only native requirements**
+
+| Feature key                              | Description                              | Since |
+|----------------------------------------|----------------------------------------|-----|
+| `http://tizen.org/feature/device_update` | Specify this key, if the application requires device update API to control the system software update of the device. | 5.0   |
```

Comment:
Question : 
@SunmLee Isn't this "device update API" is the name of API?
@sabeena21  If so, isn't "Device Update API" correct?